### DISCUSSION TITLE
"SourceFiles" in Settings.yml are relative to what?

### DISCUSSION THREAD

**Discussion by StevenTCramer** [statiqdev/Discussions#149] [Q&A] [answered]
```
SourceFiles:
- ../../Statiq.Framework/src/**/{!.git,!bin,!obj,!packages,!*.Tests,!*.Templates,}/**/*.cs
- ../../Statiq.Web/src/**/{!.git,!bin,!obj,!packages,!*.Tests,!*.Templates,}/**/*.cs
- ../../Statiq.Docs/src/**/{!.git,!bin,!obj,!packages,!*.Tests,!*.Templates,}/**/*.cs
```
From https://github.com/statiqdev/Website/blob/main/settings.yml

Notice the relative path?  But I want to know to what is it relative?

**Comment by daveaglick** [comment, marked as answer]
Great question, something I should clarify in the docs. In the case of source code, Statiq reads those files in using the `ReadFiles` module like any other file, so the root here is the virtual file system that generally consists of the `input` folder (and any other defined input folders):

![image](https://user-images.githubusercontent.com/1020407/175330367-9dd4b77b-ccfd-415f-84cd-9454c14288bf.png)

Does that answer the question? I'll add a note about this in the Statiq Docs guide.

**Reply by daveaglick** [reply]
FWIW the other kinds of Docs keys like `ProjectFiles` are also relative to the `input` folder.

**Reply by StevenTCramer** [reply]
Thanks Dave. Much appreciated.
